### PR TITLE
[fix](cloud) fix error-prone and misleading codes in cache warmup

### PR DESCRIPTION
**Pull request by freemandealer**
## Proposed changes

Issue Number: close #xxx

### CODE REVIEW

**Comment by freemandealer** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/cloud/CacheHotspotManager.java [hunk 1]
```diff
@@ -331,7 +331,7 @@ private List<Pair<TGetTopNHotPartitionsResponse, Backend>> fetchOneClusterHotSpo
         return responseList;
     }
 
-    private Long getFileCacheUsedBytes(String clusterName) throws RuntimeException {
+    private Long getFileCacheTotalBytes(String clusterName) throws RuntimeException {
```
Comment (gavinchou) [inline fe/fe-core/src/main/java/org/apache/doris/cloud/CacheHotspotManager.java:334]:
```suggestion
    private Long getFileCacheCapacity(String clusterName) throws RuntimeException {
```

File: fe/fe-core/src/main/java/org/apache/doris/cloud/CacheHotspotManager.java [hunk 2]
```diff
@@ -571,7 +521,7 @@ private Map<Long, List<Tablet>> warmUpNewClusterByTable(long jobId, String dstCl
             List<Triple<String, String, String>> tables,
             boolean isForce) throws RuntimeException {
         Map<Long, List<Tablet>> beToWarmUpTablets = new HashMap<>();
-        Long totalFileCache = getFileCacheUsedBytes(dstClusterName);
+        Long totalFileCache = getFileCacheTotalBytes(dstClusterName);
```
Comment (gavinchou) [inline fe/fe-core/src/main/java/org/apache/doris/cloud/CacheHotspotManager.java:524]:
```suggestion
        Long totalFileCache = getFileCacheCapacity(dstClusterName);
```

File: fe/fe-core/src/main/java/org/apache/doris/cloud/CacheHotspotManager.java [hunk 3]
```diff
@@ -436,7 +386,7 @@ private Map<Long, List<List<Long>>> splitBatch(Map<Long, List<Tablet>> beToWarmU
     }
 
     private Map<Long, List<Tablet>> warmUpNewClusterByCluster(String dstClusterName, String srcClusterName) {
-        Long dstTotalFileCache = getFileCacheUsedBytes(dstClusterName);
+        Long dstTotalFileCache = getFileCacheTotalBytes(dstClusterName);
```
Comment (gavinchou) [inline fe/fe-core/src/main/java/org/apache/doris/cloud/CacheHotspotManager.java:389]:
```suggestion
        Long dstTotalFileCache = getFileCacheCapacity(dstClusterName);
```

**Comment by gavinchou** [comment]
run buildall

**Review by gavinchou — APPROVED** [review]

**Review by Yukang-Lian — APPROVED** [review]

**Review by w41ter — APPROVED** [review]

**Pull request merged**